functions_framework/README.md

```diff
@@ -2,9 +2,7 @@
 
 > DISCLAIMER: this is not ready for production.
 
-|Functions Framework|Unit Tests|Lint Test|Conformance Tests|
-|---|---|---|---|
-[Dart][ff_dart]| [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
```

Comment:
Might want to delete the links too, like `ff_dart_conformance_img` in this file.

Reply:
Don't want to hold the PR up, but I personally like the idea of at least one badge to indicate the build is good -- i always like seeing that in other repos (even for library packages, not just apps). Not every visitor to the repo is aware of the main functions framework repo or even of our CI process for pull requests.